src/components/dropdown-menu/DropdownMenu.scss

```
@@ -28,15 +30,15 @@
             bottom: 0;
             left: 0;
             margin-top: 0;
+            margin-bottom: $responsive-dropdown-header-height;
             // cancels out the tether inline styling without having to set enabled=false
             transform: none !important;
         }
 
         .aria-menu:not(.submenu) {
-            position: relative;
             height: 100%;
-            margin: 0;
-            padding-top: $bdl-header-height + $bdl-grid-unit * 3;
+            margin: $responsive-dropdown-header-height 0 0 0;
```

Are we able to do something this instead
```suggestion
            margin-top: $bdl-header-height ;
```